src/businesses/apiaccounts/service/ApiAccountsService.ts

```diff
@@ -13,6 +13,19 @@ export default class ApiAccountsService {
 
   constructor(private readonly client: RequestClientWrapper) {}
 
+  /** 
+   * Do Not Use. This is for internal use only
+   * 
+   */
+  public getById(apiAccountId: string) {
```

Comment:
The auth on this one is weird, so this is something that can only be used internally. Not sure of a better way to do this.  The reason i added here is so that we can use it and get all retry logic and things like that is already built in here.